src/legacy/core_plugins/kibana/public/index.scss

```diff
@@ -7,12 +7,3 @@
 // Public UI styles
 @import 'src/legacy/ui/public/index';
 
-// Has to come after visualize because of some
-// bad cascading in the Editor layout
-@import '../../../../plugins/maps_legacy/public/index';
```

Comment:
You're removing this import does that mean it's imported elsewhere?

Reply:
That was an oversight, thanks for noticing. I moved the import into the Kibana platform plugin.